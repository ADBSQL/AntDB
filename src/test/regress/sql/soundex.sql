set grammar to oracle;
select soundex('moon') from dual;
select soundex('while') from dual;
select soundex('ask') from dual;
select soundex('basket') from dual;
select soundex('yow') from dual;
select soundex('lll') from dual;
select soundex('dt') from dual;
select soundex('bcdlm') from dual;
select soundex(to_char('ask')) from dual;
select soundex('') from dual;
select soundex('NaN') from dual;
select soundex(null) from dual;
