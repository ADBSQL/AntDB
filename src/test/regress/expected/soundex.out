set grammar to oracle;
select soundex('moon') from dual;
 soundex 
---------
 M500
(1 row)

select soundex('while') from dual;
 soundex 
---------
 W400
(1 row)

select soundex('ask') from dual;
 soundex 
---------
 A200
(1 row)

select soundex('basket') from dual;
 soundex 
---------
 B230
(1 row)

select soundex('yow') from dual;
 soundex 
---------
 Y000
(1 row)

select soundex('lll') from dual;
 soundex 
---------
 L400
(1 row)

select soundex('dt') from dual;
 soundex 
---------
 D300
(1 row)

select soundex('bcdlm') from dual;
 soundex 
---------
 B234
(1 row)

select soundex(to_char('ask')) from dual;
 soundex 
---------
 A200
(1 row)

select soundex('') from dual;
 soundex 
---------
 
(1 row)

select soundex('NaN') from dual;
 soundex 
---------
 N500
(1 row)

select soundex(null) from dual;
 soundex 
---------
 
(1 row)

